audit_math/sampler.py

```diff
@@ -62,7 +62,6 @@ def draw_sample(
 def draw_ppeb_sample(
     seed: str,
     contest: Contest,
-    manifest: Dict[str, int],
```

Comment:
@umbernhard this argument was unused, and there wasn't any obvious plan to use it, so I removed it. Seem okay?

Reply:
No, that may be a bug, lemme look.

Reply:
oh, I see, I'm using the batch results instead of the manifest. Gonna drop a convo in slack about this.